build.psm1

```diff
@@ -3,7 +3,7 @@
 $script:TestModulePathSeparator = [System.IO.Path]::PathSeparator
 
 $dotnetCLIChannel = "release"
-$dotnetCLIRequiredVersion = "2.0.0"
+$dotnetCLIRequiredVersion = "2.0.2"
```

Comment:
@daxian-dbw While you are at it... maybe it is a good idea to change the required version check from "exact match" to "only major+minor version should match" in build.psm1:414 . See #5260 or more details.

Reply:
@anmenaga As far as I know, this issue has been addressed by the PR #5118. Please see the original issue #5015.
`global.json` defines which version of SDK we want to use, so it will work even if you have 2.0.2 installed.